vars/main.yml

```diff
@@ -0,0 +1,29 @@
+restund_all_public_ipv4s: "{{ ansible_facts.all_ipv4_addresses | ipaddr('public') }}"
+restund_all_private_ipv4s: "{{ ansible_facts.all_ipv4_addresses | ipaddr('private') }}"
+
+# INFO: Lists of private IP ranges to help setting up firewall rules and prevent exposure of private services
+# DOCS: https://www.rtcsec.com/post/2021/01/details-about-cve-2020-26262-bypass-of-coturns-default-access-control-protection/#further-concerns-what-else
+restund_blocked_ipv4_network_ranges:
+  - '0.0.0.0/8'
+  - '10.0.0.0/8'
+  - '100.64.0.0/10'
+  - '127.0.0.0/8'
```

Comment:
We should remove `127.0.0.0` here otherwise `rex` can't reach the status module, and DNS will also break (Is hosted on `127.0.0.53` on ubuntu)

Reply:
Good point. I had similar reservations. Addressed: https://github.com/wireapp/ansible-restund/pull/18/commits/18c78c0cf8a9f5988e3946160704557a0ddefe17. Let me know if the commit message is misleading.